doc/unet.txt

```diff
@@ -12,56 +15,80 @@ Its goal is then to predict each pixel's class.
 Compared to image classification, the difficulty arise from the fact that localisation is important.
 The network must capture the overall context and recognize very precise details.
```

Comment:
This may be a bit redundant with the tutorial on segmentation, which is already a pre-requisite.
Maybe just refer to the FCN again here, and say something about the difference (I know it is not obvious, given you do that in the Model section just after).